app/models/course_enrichment.rb

```diff
@@ -32,7 +32,7 @@ class CourseEnrichment < ApplicationRecord
                  interview_process: [:string, store_key: 'InterviewProcess'],
                  other_requirements: [:string, store_key: 'OtherRequirements'],
                  personal_qualities: [:string, store_key: 'PersonalQualities'],
-                 qualifications: [:string, store_key: 'Qualifications'],
+                 required_qualifications: [:string, store_key: 'RequiredQualifications'],
```

Comment:
https://github.com/DFE-Digital/manage-courses-api/blob/7419131f3fb92e41c8f131281afaba96e2abb1ad/src/ManageCourses.Api/Model/CourseEnrichmentModel.cs#L16

are you sure?